### ISSUE TITLE
BioModel Discussion Issue for eos92sw

### ISSUE THREAD

**Issue by Zainab-ik** [ersilia-os/eos92sw#12]
**Summary !!!**

- **BioModel Name**: Pu2019 - eToxPred: an ML-based approach to estimate the toxicity, and synthetic accessibility of drug candidates

- **BioModel Tag**: Machine learning model, Ersilia, FAIR AIML

- **Metadata Comments**: 
The eToxPred tool has been developed to predict how easy it is to make the molecule in the laboratory, and the probability of the molecule being toxic to humans.

1. Can I comfirm if this is a Neural network model? there are mentions of nodes, and layers, and the type of algorithm. 
2. It's difficult to identify the exact training dataset. It's a combination of data from several database. Should we list them all, or how do we consolidate that. 
3. All these algorithm were mentioned; Restricted Boltzmann Machine (RBM), Deep Belief Network (DBN), Extremely Randomized Trees or Extra Trees (ET), algorithm. However, there was more emphasis on ET algorithm. 
4. Both classification and regression task evaluation was done. In the Ersilia Repository, there was only a mention of regression as task, should we stick to that for the evaluation metric? 

- **Contributor**:
> 
 | roles             | Names | 
 | :---------------- | :------: | 
 | Curator        |   @Zainab-ik   | 
 | Code contributor      |   @miquelduranfrigola      | 
 | Reviewer       |   @GemmaTuron    | 

- **Curation status** :  manually curated

- **Annotation file**: [here](https://docs.google.com/spreadsheets/d/1yeDr-kJpzMIiY-wrUdDWNMCY7FbD_rBSxCdP19ZgG60/edit?usp=sharing)

**Comment by Zainab-ik** [comment]
All databases in table 1 added. Annotation complete and ready for review.

**Comment by GemmaTuron** [comment]
Good job @Zainab-ik !

Looks good to go ahead
